src/gdphelper/gdpplotter.py

```diff
@@ -1,26 +1,84 @@
 import matplotlib.pyplot as plt
+import pandas as pd
+import numpy as np
 
-
-def gdpplotter(type_chart, type_feature):
+def gdpplotter(data_name, aggregation):
     """Plot a line chart of periods and the selected feature.
     
     Parameters
     ----------
-    type_chart : string
-        There are two types of chart, "all_together" or "split_charts".
-        The first type will bring up all sub-features in the same plot
-        The second type will bring up all one chart for each sub-feature.
-    type_feature : string
-        There are two types of features, "region" and "industry".
-        The first one will break down the GDP by each reageion.
-        The second one will break down the GDP by each industry.
-
+    data_name : string
+        This is the name of the dataset that you want to draw a chart.
+    aggregation : string
+        There are two types of aggregation, "canada" and "province".
+        The first one will give the GDP by total Canada.
+        The first one will give the GDP by province.
+        
     Returns
     -------
-        line chart of GDP by the periods within the dataset.
+        line chart of GDP by the periods.
 
     Examples
     --------
-    >>> 
-
+    >>>
     """
+    # Checking the inputs of the function: data_name
+    if type(data_name) == int:
+        pass
+    else:
+        raise ValueError("data_name should be the NUMBER of your downloaded databese")
+    
+    # Checking the inputs of the function: aggregation
+    if aggregation in ["canada", "province"]:
+        pass
+    else:
+        raise ValueError(
+            'The aggragation should be "canada" or "province".'
+        )
+
+
+    #reading the database within the function
+    file = f"{data_name}.csv"
+    df = pd.read_csv(file)
+    print("database loaded")
```

Comment:
As Gabe mentioned, this function does not need to read data from files as this is done in upstream functions. 😸